src/jobs/lint.yml

```diff
@@ -3,6 +3,14 @@ description: |
 
 docker:
   - image: cimg/python:3.10.2
+resource_class: << parameters.resource-class >>
+
+parameters:
+  resource-class:
+    description: Support setting / overriding executor resource class
+    type: enum
+    enum: ["small", "medium", "medium+", "large", "xlarge", "2xlarge", "2xlarge+"]
```

Comment:
This would still prevent use of self-hosted runners, and would need adjustment if Circle added new resource classes, but I think this still probably is better in terms of providing some validation? happy to adjust if needed, though.